private-templates-service/server/src/app.ts

```diff
@@ -20,6 +22,18 @@ app.use(passport.initialize());
 app.use(passport.session());
 app.use(bodyParser.json());
 app.use(bodyParser.urlencoded({ extended: true }));
+app.use(helmet.hsts({
+  maxAge: 15552000
+}));
+app.use(session({
+  secret: '#{CLIENT_ID_TOKEN}#',
+  cookie: {
```

Comment:
is this a cookie we need compliance for

Reply:
Yeah this is for if we do use cookies to only use secure ones